tests/publiccloud/create_aistack_env.pm

```diff
@@ -93,45 +98,57 @@ sub config_kubectl {
 }
 
 sub install_aistack_chart {
-    my ($instance, $ai_chart_repo, $namespace, $vf_name) = @_;
+    my ($instance, $namespace) = @_;
     my $SECRET_application_collection = get_var('_SECRET_DOCKER');
     my $cert_repo = get_var('HELM_CERTS');
     my $ingress_repo = get_var('HELM_INGRESS');
     my $docker_user_name = get_var('USER_DOCKER');
     my $repo_url = get_var('HELM_CHARTS');
-    my $git_token = get_var('_SECRET_SSH');
     my $ing_ver = get_var('ING_VERSION');
     my $local_storage_name = 'local_path_storage.yaml';
+    my $milvus_custom = 'milvus_custom.yaml';
+    my $milvus_values = 'milvus_values.yaml';
+    my $ollama_values = 'ollama_values.yaml';
+    my $openwebui_values = 'open_webui_values.yaml';
+    my $milvus_helm_repo = 'oci://dp.apps.rancher.io/charts/milvus';
+    my $ollama_helm_repo = 'oci://dp.apps.rancher.io/charts/ollama';
+    my $openwebui_helm_repo = 'oci://dp.apps.rancher.io/charts/open-webui';
 
     record_info('AISTACK charts install');
     assert_script_run("helm list --all-namespaces");
     assert_script_run("kubectl get pods --all-namespaces");
 
     # Access to Application collection registery
-    # Get docker username and password
+    # local-path-storage.yaml is a copy off https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.28/deploy/local-path-storage.yaml
     assert_script_run("kubectl create ns $namespace");
     assert_script_run("kubectl create secret docker-registry application-collection --docker-server=dp.apps.rancher.io --docker-username='$docker_user_name' --docker-password='$SECRET_application_collection' -n $namespace", timeout => 120);
```

Comment:
PLease consider that this command the PASSWORD is reported in **CLEAR** in the log, see:
https://openqa.suse.de/tests/16182166#step/create_aistack_env/142
I'd suggest to found a way to avoid to pass `sensible` parameters to commands appearing in logs.